File: wonderbuild/benchmarks/misc/stat/stat.cpp
#include <sys/stat.h>
#include <iostream>
#include <errno.h>

int main() {
	time_t max_time;
	struct stat st;
	for(unsigned int i(0); i < 1000000; ++i) {
		if(stat(".", &st) != 0) {
			std::cerr << errno << '\n';
			return errno;
		}
		if(st.st_mtime > max_time) max_time = st.st_mtime;
	}
	std::cout << max_time << '\n';
	return 0;
}

File: wonderbuild/benchmarks/misc/stat/stat.py
#! /usr/bin/env python

if __name__ == '__main__':
	import sys, os, stat, time
	max_time = time.time()
	for i in xrange(1000000):
		try: st = os.stat('.')
		except OSError: sys.exit(1)
		t = st.st_mtime
		if t > max_time: max_time = t
	print max_time		
